packages/components/src/components/primitives/Button/Button.tsx

```diff
@@ -0,0 +1,5 @@
+import React, { FC } from 'react';
+import { Button as ChakraButton } from '@chakra-ui/react';
+import { ButtonProps } from './types';
+
+export const Button: FC<ButtonProps> = (props) => <ChakraButton {...props} />;
```

Comment:
please re-export this component in https://github.com/cmpsr/composer/blob/with-chakra/packages/components/src/components/primitives/index.ts